frontend/src/features/admin-form/create/workflow/components/WorkflowContent/EditStepBlock/RespondentBlock.tsx

```diff
@@ -119,158 +290,32 @@ export const RespondentBlock = ({
         </>
       ) : (
         <>
-          <Text textStyle="subhead-3">Respondent in this step</Text>
           <FormControl
             isReadOnly={isLoading}
-            id="workflowType"
```

Comment:
Might not need `id` anymore since there's no confusion by react from `<RespondentInput />`

Reply:
removed!